src/review/dto/create-review.dto.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  Min,
+  Max,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateReviewDto {
+  @IsNotEmpty()
+  @IsNumber()
+  @ApiProperty({
+    description: 'ID of the guide being reviewed',
+    example: 1,
+  })
+  guideId: number;
+
+  @IsNotEmpty()
+  @IsNumber()
+  @Min(1)
+  @Max(5)
+  @ApiProperty({
+    description: 'Rating given by the user (1-5 stars)',
+    example: 5,
+  })
+  stars: number;
```

Comment:
Why not just call it rating?

Reply:
I think that stars is a better naming, it is more explanatory

Reply:
Its really not. This could be squares, list or whatever. what if we change it in the future from stars to something else?